Add information on configuring the thread pool size (#18901)
ballerina-platform/ballerina-lang: stdlib/jdbc/src/main/ballerina/src/java.jdbc/Module.md

creating tables and executing stored procedures. It also supports selecting, inserting, deleting, updating, and batch 
updating data. Samples for these operations can be found below. 

> >**Note:** The JDBC client module is not designed to use the Ballerina non-blocking thread model. Hence, the default thread pool size used in Ballerina (i.e., number of processers available * 2) may not deliver the optimal performance. Therefore, you can configure the thread pool size using the `BALLERINA_MAX_POOL_SIZE` system variable as required.

Reviewer: ```suggestion
>**Note:** The JDBC client module is not designed to use the Ballerina non-blocking thread model. Hence, the default thread pool size used in Ballerina (i.e., number of processers available * 2) may not deliver the optimal performance. Therefore, you can configure the thread pool size using the `BALLERINA_MAX_POOL_SIZE` environment variable as required.
```

> >**Note:** The JDBC client module is not designed to use the Ballerina non-blocking thread model. Hence, the default thread pool size used in Ballerina (i.e., number of processers available * 2) may not deliver the optimal performance. Therefore, you can configure the thread pool size using the `BALLERINA_MAX_POOL_SIZE` environment variable as required.

Reviewer 2: ```suggestion
>**Note:** The JDBC client module is not designed to use the Ballerina non-blocking thread model. The default thread pool size used in Ballerina is number of processers available * 2. You can configure the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.
```

    error err = retCall;
    io:println("Stored procedure call failed: ", <string>err.detail()["message"]);
}
```

> >**Note:** The default thread pool size used in Ballerina is number of processers available * 2. You can configure the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.

Reviewer 3: I think it's better the "processers" here should be spelled as "[processors](https://www.intel.com/content/www/us/en/products/processors.html)".
